modules/wmco-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.adoc
+// * windows_containers/creating_windows_machinesets/creating-windows-machineset-azure.adoc
+// * windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.adoc
+// * windows_containers/creating_windows_machinesets/creating-windows-machineset-gcp.adoc
+// * windows_containers/creating_windows_machinesets/creating-windows-machineset-nutanix.adoc
```

Comment:
I'd publish this under enabling-windows-container-workloads, instead of the machineSet section that is platform specific. 

https://75056--ocpdocs-pr.netlify.app/openshift-enterprise/latest/windows_containers/enabling-windows-container-workloads

Reply:
@jrvaldes I agree that the new section should go there. But at that point, we don't have any Windows nodes. Does the user need the Windows nodes in order to  update the `DefaultWebProxy` and create the PowerShell profile script?